packages/react-conformance/src/defaultTests.tsx

```diff
@@ -65,12 +67,95 @@ export const defaultTests: TestObject = {
   },
 
   /** Constructor/component name matches filename */
-  'name-matches-filename': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
+  'name-matches-filename': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
     it(`Component/constructor name matches filename`, () => {
       const { componentPath, displayName } = testInfo;
       const fileName = path.basename(componentPath, path.extname(componentPath));
 
       expect(displayName).toEqual(fileName);
     });
   },
+
+  /** Ensures component is exported at top level allowing import { Component } from 'packageName' */
+  'exported-top-level': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    if (!testInfo.isInternal) {
+      it(`is exported at top-level`, () => {
+        const { displayName, componentPath, Component } = testInfo;
+        const rootPath = componentPath.replace(/src.*/, '');
```

Comment:
To be safe you should make this a little more specific (same in other places):
```suggestion
        const rootPath = componentPath.replace(/[\\/]src[\\/].*/, '');
```